.rubocop_todo.yml

```diff
@@ -0,0 +1,601 @@
+# This configuration was generated by `rubocop --auto-gen-config`
+# on 2014-05-26 15:23:01 +0100 using RuboCop version 0.22.0.
```

Comment:
Looks like you generated the todo file (and also the main .yml) with the wrong version of fog, all the cop names have been changed.

Reply:
I didn't generate it at all. I copied it from `fog/fog`'s master branch to keep the styling consistent and then renamed the ones rubocop complained about